sdk/ai/Azure.AI.Inference/samples/Sample8_ChatCompletionsWithTelemetry.md

```diff
@@ -0,0 +1,127 @@
+
+## Project creation and dependency installation
+First, we will create the console application project and add `Azure.AI.Inference` as a dependency. The first command will create the project called `TelemetryDemo.csproj`. The `dotnet add package <…>` command will modify project file and in future we will need to run `dotnet restore` to install all dependencies, if we will remove the installed ones.
+
+```dotnetcli
+dotnet new console --name TelemetryDemo --output TelemetryDemo
+dotnet add package Azure.AI.Inference --prerelease
+```
+
+Now we will need to add the dependencies for OpenTelemetry.
+
+```dotnetcli
+dotnet add package OpenTelemetry
+dotnet add package OpenTelemetry.Exporter.Console
+dotnet add package OpenTelemetry.Exporter.OpenTelemetryProtocol
+dotnet add package Azure.Monitor.OpenTelemetry.AspNetCore --prerelease
+```
+
+## Create the simple application with telemetry
+The `dotnet new` created the project with the single file called Program.cs. Let us edit this file with the IDE of choice.
+
+First we will import open telemetry and Azure.AI.Inference
+```C# Snippet:Azure_AI_Inference_TelemetrySyncScenario_import
+//Azure imports
+using Azure;
+using Azure.AI.Inference;
+// Open telemetry imports
+using OpenTelemetry;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Trace;
+using Azure.Monitor.OpenTelemetry.Exporter;
+using OpenTelemetry.Metrics;
+```
+
+In this exmple, we will log metrics and events to console, to Azure Monitor using Application Insights and will use the Open Telemetry protocal to log data into Aspire Dashboard see the [installation instructions](https://learn.microsoft.com/dotnet/aspire/fundamentals/dashboard/standalone?tabs=bash). Data ingestion into Aspire Dashboard does not require Azue account.
+Now we will define the `endpoint`, `credential` and `model`. To log metrics and events to [Application Insights](https://learn.microsoft.com/azure/azure-monitor/app/app-insights-overview) we will need to get the connection string. Please open the Azure portal, create the Application Insights resource you wish to use for storing the telemetry. After that open the main page and find the "Connection String". It generally will have the format similar to "InstrumentationKey=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx;IngestionEndpoint=https://region-x.in.applicationinsights.azure.com/;LiveEndpoint=https://eastus.livediagnostics.monitor.azure.com/;ApplicationId=xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx". In our code we will use the connection string from environment variable.
+```C# Snippet:Azure_AI_Inference_TelemetrySyncScenario_variables
+var endpoint = new Uri(System.Environment.GetEnvironmentVariable("MODEL_ENDPOINT"));
+var credential = new AzureKeyCredential(System.Environment.GetEnvironmentVariable("GITHUB_TOKEN"));
+var model = System.Environment.GetEnvironmentVariable("MODEL_NAME");
```

Comment:
MODEL_NAME is the model deployment name when model is deployed with Azure AI Studio?

Also here, do we have common naming for Azure AI Model Deployment Name

Reply:
Yes, in this case MODEL_NAME is a deployed model name in the Azure. We are using it in our samples; however, I think, we do not have the standard for it.